src/languages/english.properties

```diff
@@ -45,15 +45,15 @@ GROUP_HAS_PERMISSIONS=The group '%s' has the following permissions: %s
 USER_HAS_PERMISSIONS=The player '%s' has the following permissions: %s
 GROUP_NO_SPECIFIC_PERMISSIONS=The group '%s' has no direct permissions.
 USER_NO_SPECIFIC_PERMISSIONS=The player '%s' has no direct permissions.
-ERROR_SAME_PERMISSIONS_OR_HIGHER=You can't modify a player the with same permissions as you, or higher.
+ERROR_SAME_PERMISSIONS_OR_HIGHER=You can't modify a player with same permissions as you, or higher.

```

Comment:
You can't modify a player with the same permissions as you, or higher

Reply:
In my opinion, what I already have is correct, but I will change to what you say anyway